File: Modul_4/Soal_5/PRAK405-2310817210015-MuhammadBukhariFitri.py
input = input().split()

angka1, angka2 = map(int, input)

hasilPerkalian = 0
hasilLooping = 0
for i in range (1, angka1 + 1) :
    perkalian = i * angka2
    hasilPerkalian += perkalian
    if(i > 1) :
        for i in range (i, 1, -1) :
            print(f"({i} * {angka2}) + ", end="")
        print(f"({1} * {angka2})", end="")
        print(f" = {hasilPerkalian}")

    elif(i == 1) :
        print(f"({i} * {angka2})", end="")
        print(f" = {hasilPerkalian}")

    hasilLooping += hasilPerkalian
print(f"{hasilLooping}")

File: Modul_4/Soal_5/PRAK405-2310817210015-MuhammadBukhariFitri.c
#include <stdio.h>

int main(void) {
    int angka1, angka2, hasilPerkalian = 0, hasilLooping = 0;

    scanf("%d %d", &angka1, &angka2);
    for(int i = 1; i <= angka1; i++) {
        int perkalian = i * angka2;
        hasilPerkalian += perkalian;
        if(i > 1) {
            for(int j = i; j > 1; j--) {
                printf("(%d * %d) + ", j, angka2);
            }
            printf("(%d * %d) ", 1, angka2);
            printf("= %d\n", hasilPerkalian);
        }
        else if(i == 1) {
            printf("(%d * %d) ", i, angka2);
            printf("= %d\n", hasilPerkalian);
        }
        hasilLooping += hasilPerkalian;

    }
    printf("%d ", hasilLooping);

}